example/implementation/fasthttp/README.md

```
@@ -0,0 +1,236 @@
+
+| Type                    | Description                                                                        |
+|-------------------------|------------------------------------------------------------------------------------|
+| `respError`             | JSON response body with an `Error` string field                                    |
+| `HttpError`             | Error carrying an HTTP status code, used to communicate status between middlewares |
+| `validator`             | Interface with `Validate() error`, checked by ValidationServerMiddleware           |
+| `FastHTTPHeaderCarrier` | Adapts `*fasthttp.Request` headers to OTel `propagation.TextMapCarrier`            |
+
+## Server Middlewares
+
+### Default chain (`GetServerMiddlewares`)
+
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)